### PR TITLE
Accordion.

### PR DESCRIPTION
**Pull request by SebastianStehle**
I thought that we are almost complete. Makes no sense to stop at 99% ;)

### CODE REVIEW

File: Mjml.Net/Components/Body/AccordionElementComponent.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
+using Mjml.Net.Internal;
+using System.Xml;
+
```
Comment (LiamRiddell) [inline Mjml.Net/Components/Body/AccordionElementComponent.cs:3]: Can we fix the build warning for consistency?

File: Mjml.Net/Components/Body/AccordionComponent.cs [hunk 2]
```diff
@@ -0,0 +1,127 @@
+using Mjml.Net.Helpers;
+
+namespace Mjml.Net.Components.Body
+{
+    public partial class AccordionComponent : BodyComponentBase
+    {
+        public override string ComponentName => "mj-accordion";
+
```
Comment (LiamRiddell) [inline Mjml.Net/Components/Body/AccordionComponent.cs:8]: Implement allowed parents once #92 is merged.

**Review by LiamRiddell — CHANGES_REQUESTED** [review]
Everything looks good but I think we need to merge #92 into main and then get these components updated with the new AllowedParents property.

File: Mjml.Net/Components/Body/AccordionElementComponent.cs (hunk 1 above)
Comment (SebastianStehle) [reply Mjml.Net/Components/Body/AccordionElementComponent.cs:3]: Sure

**Pull request merged**